gradle-plugins/src/main/kotlin/io/opentelemetry/javaagent/muzzle/matcher/MuzzleGradlePluginUtil.kt

```diff
@@ -65,7 +67,9 @@ class MuzzleGradlePluginUtil {
         .invoke(null, userClassLoader, assertPass)
         as Map<String, List<Any>>
 
-      allMismatches.forEach { moduleName, mismatches ->
+      allMismatches.filterKeys {
+        !excludedInstrumentationModules.contains(it)
```

Comment:
@mateuszrzeszutek were you thinking of instantiate the class `it` in order to check its instrumentation name? or were you thinking of suppressing the instrumentation ahead of time using the instrumentation name suppression?

Reply:
I was thinking on passing the set of excluded instrumentation names over to `ClassLoaderMatcher` (which instantiates the `InstrumentationModule`s) and excluding them there. I don't think that suppressing them ahead of time would work, we ignore the enabled flag in the muzzle plugin.